### ISSUE TITLE
failed to MapImage after win10 1809 update

### ISSUE THREAD

**Issue by ollydbgz** [DarthTon/Blackbone#300]
when
`process.mmap().MapImage(path, flags);`

get an error from Utils::GetErrorDescription(image.status)
```
0xC0000138
{Ordinal Not Found}
The ordinal %ld could not be located in the dynamic link library %hs.
```

win10 version 1809 OS build 17763.1

**Comment by Chopard** [comment]
Who fixed the problem?

**Comment by LeimonDust** [comment]
Any news?

**Comment by DarthTon** [comment]
I'm waiting for 1809 public re-release to add support.

**Comment by nanoric** [comment]
> I'm waiting for 1809 public re-release to add support.

I'm a newbee. May I ask what "public re-release" is ?

**Comment by DarthTon** [comment]
When update becomes available again through windows update for normal users.

**Comment by LeimonDust** [comment]
> When update becomes available again through windows update for normal users.
As far as I know re release took place
https://support.microsoft.com/en-us/help/4464619/windows-10-update-history

**Comment by DarthTon** [comment]
Fixed in 96b12708cb5bbc24851e6fefdfd317beec71f779
